### PR TITLE
build: no ssr for local development

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Being able to install and use local references to nns-js, sns-js, etc.

# Changes

- disable `ssr` for local development - and development only - until we solve issue https://github.com/dfinity/ic-js/issues/238.

### CODE REVIEW

File: frontend/src/routes/+layout.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 export const prerender = true;
+// TODO: no ssr for local development until https://github.com/dfinity/ic-js/issues/238 solved
+export const ssr = import.meta.env.PROD;
```
Comment (lmuntaner) [inline frontend/src/routes/+layout.ts:3]: Did you try it? I tried also setting `ssr = false` but it wasn't working for me.

Comment (peterpeterparker) [reply frontend/src/routes/+layout.ts:3]: Generally I try before providing a PR 😉. Yes I did, it works out. 

Maybe you did not set the `ssr = false` within the main `+layout.ts` file?

Comment (lmuntaner) [reply frontend/src/routes/+layout.ts:3]: maybe... weird

Comment (peterpeterparker) [reply frontend/src/routes/+layout.ts:3]: so can you accept the PR?

Comment (lmuntaner) [reply frontend/src/routes/+layout.ts:3]: oops sorry! I cancelled the other one and forgot to approve this one

**Review by lmuntaner — APPROVED** [review]

File: frontend/src/routes/+layout.ts (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/routes/+layout.ts:3]: no worries at all, thanks!

**Pull request merged**